### PR TITLE
v6.3: add sm4 for tiflash to encryption-at-rest.md

### PR DESCRIPTION
**Pull request by lidezhu**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: encryption-at-rest.md [hunk 1]
```diff
@@ -36,6 +36,8 @@ TiFlash 支持静态加密。数据密钥由 TiFlash 生成。TiFlash（包括 T
 
 如果 TiFlash 中部署了 Grafana 组件，可以查看 **TiFlash-Proxy-Details** -> **Encryption**。
 
+SM4 加密只在 v6.3.0 及之后版本的 TiFlash 上支持。v6.3.0 之前的 TiFlash 仅支持 AES 加密。
```
Comment (ran-huang) [inline encryption-at-rest.md:39]: 这里缺少上下文。建议参考 TiKV 的写法（L23）：
> TiKV 支持静态加密，即在 CTR 模式下使用 AES 或 SM4 对数据文件进行透明加密。

**Review by ran-huang — APPROVED** [review]
LGTM

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Comment by ran-huang** [comment]
@flowbehappy PTAL

**Review by flowbehappy — APPROVED** [review]

**Pull request merged**